Jenkinsfile

```diff
@@ -1,134 +1,198 @@
-def cleanUp() {
-    sh '''#!/bin/bash
-    pkill -f app.js || true
-    bash ~/lisk-test/lisk.sh stop
-    pkill -f selenium -9 || true
-    pkill -f Xvfb -9 || true
-    rm -rf /tmp/.X0-lock || true
-    pkill -f webpack -9 || true
+def fail(reason) {
+  sh '''
+  ps aux > ps_log.log
+  netstat -tunap > netstat_log.log
   '''
-  deleteDir()
+
+  def pr_branch = ''
+  if (env.CHANGE_BRANCH != null) {
+    pr_branch = " (${env.CHANGE_BRANCH})"
+  }
+  slackSend color: 'danger', message: "Build #${env.BUILD_NUMBER} of <${env.BUILD_URL}|${env.JOB_NAME}>${pr_branch} failed (<${env.BUILD_URL}/console|console>, <${env.BUILD_URL}/changes|changes>)\nCause: ${reason}", channel: '#lisk-explorer-jenkins'
+  currentBuild.result = 'FAILURE'
+  error("${reason}")
 }
 
 node('lisk-explorer-01'){
-  lock(resource: "lisk-explorer-01", inversePrecedence: true) {
+  try {
     stage ('Prepare Workspace') {
-      cleanUp()
+      deleteDir()
       checkout scm
     }
 
     stage ('Build Dependencies') {
       try {
-        sh '''#!/bin/bash
+        sh '''
 
         # Install Deps
         npm install
+        ./node_modules/protractor/bin/webdriver-manager update
+        wget https://downloads.liskwallet.net/lisk/redis/redis-3.2.9-Linux-x86_64.tar.gz
```

Comment:
Please just upload the binaries to the jenkins machine instead of dowloading every time.